docs/UserGuide.md

```diff
@@ -135,22 +138,31 @@ Examples:
 * `list` followed by `delete 2` deletes the 2nd student in the application.
 * `find Betsy` followed by `delete 1` deletes the 1st student in the results of the `find` command.
 
-#### Clearing all entries : `clear`
+#### Clearing all students : `clear`
 
-Clears all entries from the application.
+Clears all students from the application.
 
 Format: `clear`
 
-### Managing Classes: [Coming soon...]
+### Managing Classes: [coming soon]
 
 #### Adding a class: `add-class`
 
 Adds a class to the application.
 
-Format: `add-class n/NAME`
+Format: `add-class [n/CLASS_NAME] [m/MODULE_CODE]`
 
 Examples:
-* `add-class n/CS2013T Tutorial T10`
+* `add-class n/Tutorial T10 m/CS2013T`
```

Comment:
I don't think we are supporting the separation of `CLASS_NAME` from `MODULE_CODE`?

Reply:
Resolved by removing MODULE_CODE